app/src/main/java/ch/epfl/sdp/blindwar/domain/game/GameSolo.kt

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.sdp.blindwar.domain.game
+
```

Comment:
Why is there a whole class commented out ?  You should remove it if it is not necessary.